src/main.rs

```diff
@@ -47,6 +48,7 @@ fn main() {
         Some("vagga_version") => version::main(),
         Some("vagga_wrapper") => wrapper::main(),
         Some("vagga_build") => builder::main(),
+        Some("vagga_pack") => pack::main(),
```

Comment:
I don't think we should add more variants here. Is there any lightweight alternatives? Do you actually need to spawn a process to perform the task?

This was just a legacy when migrating from vagga 0.2  (which contained multiple binaries) to vagga 0.3 (which has single binary for everything)